src/noisy_function.jl

```diff
@@ -43,3 +43,34 @@ function noisy_poly{T<:Real,R<:Real}(coef::AbstractVector{R}, x::AbstractVector{
     y .+= noise .* f_rand(n)
     x_vec, y
 end
+
+
+"""
+`x_one, y_one, x_zero, y_zero = noisy_spiral(n, a, theta, b; noise = 0.01, f_rand = randn)`
+
+Generates `n` noisy responses for a spiral with two labels. Uses the radius, angle
+and scaling arguments to space the points in 2D space and adding `noise .* f_randn(n)`
+to the response.
+"""
+function noisy_spiral(n::Int = 96, a::Real = 6.5, theta::Real = 16.0, b::Real=104.0; noise::Real = 0.1, f_rand::Function = randn)
```

Comment:
The original code has `for (i=0; i<=96; i++) `, which includes one more element since it starts at 0. Should we reflect this by counting to 97?